types/which-polygon/package.json

```diff
@@ -7,7 +7,7 @@
     ],
     "dependencies": {
         "@types/geojson": "*",
-        "@types/rbush": "*"
+        "@types/rbush": "^3"
```

Comment:
Same remark as polygon-lookup, the underlying library is actually using rbush 2.x.